Dockerfile

```diff
@@ -1,25 +1,28 @@
-# Stage 1 - the build react app
-FROM node:12.4.0-alpine as build-deps
+# Stage 1 - install the heavy dependencies
+FROM --platform=$BUILDPLATFORM node:lts-alpine as build-deps
 WORKDIR /usr/src/app
-COPY client/package.json client/package-lock.json ./
-RUN npm i
+COPY client/package.json ./
```

Comment:
My understanding is that `package-lock.json` would be important here so there are no surprises caused by the final build using a different version of some package than was used during development. Is there a reason you feel this is undesirable?

Reply:
Good catch, I actually overlocked that - will fix